pkg/interface/src/apps/chat/components/lib/content/text.js

```diff
@@ -3,7 +3,6 @@ import { Link } from 'react-router-dom';
 import ReactMarkdown from 'react-markdown';
 import RemarkDisableTokenizers from 'remark-disable-tokenizers';
 import urbitOb from 'urbit-ob';
-import { Link } from 'react-router-dom'; 
```

Comment:
This was preventing a compile?

Reply:
I fixed the missing Link import in my branch as well, so the two imports of the same thing made it impossible to compile.